### PR TITLE
Move ngeox check to a dedicated rule

### PR DESCRIPTION
**Pull request by gberaudo**
Follow up of #2744.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -223,7 +226,7 @@ eof-newline:
 	buildtools/test-eof-newline
 
 .PHONY: test
-test: .build/ol-deps.js .build/ngeo-deps.js .build/gmf-deps.js .build/templatecache.js .build/gmftemplatecache.js .build/node_modules.timestamp .build/examples-hosted/lib/proj4.js
+test: .build/ol-deps.js .build/ngeo-deps.js .build/gmf-deps.js .build/templatecache.js .build/gmftemplatecache.js .build/node_modules.timestamp .build/examples-hosted/lib/proj4.js check-ngeox
```
Comment (sbrunner) [inline Makefile]: Why adding that here, for me it should be added in `check`, and this
```
- if [ "${DO_LINT}" = TRUE ]; then make check-ngeox; fi
```
should be added in `.travis.yaml` https://github.com/camptocamp/ngeo/blob/master/.travis.yml#L105

Comment (gberaudo) [inline Makefile]: OK, done

**Comment by sbrunner** [comment]
UP...

**Comment by gberaudo** [comment]
... and click DOWN. ^^

**Pull request merged**